src/Post_Meta/Meta_Map.php

```diff
@@ -52,12 +53,13 @@ public function get_keys() {
 	/**
 	 * @param int $post_id
 	 * @param string $key
-	 * @return mixed The value for the given key
+	 * @return mixed The value for the given key or null
```

Comment:
```suggestion
	 * @return mixed The value for the given key
```
The mixed pseudo type covers null already.